atomsci/ddm/pipeline/transformations.py

```diff
@@ -226,7 +220,8 @@ def __init__(self,
             self.transform_gradients = transform_gradients
             self.move_mean = move_mean
             if self.transform_gradients:
-                true_grad, ydely_means = get_grad_statistics(dataset)
+                # TODO: Figure out where get_grad_statistics is defined
+                true_grad, ydely_means = get_grad_statistics(dataset) # noqa: F821
```

Comment:
We weren't able to resolve this issue, so we ignored it for now, but maybe someone from the team can help us identify where `get_grad_statistics` should be imported from or defined

Reply:
I think I found a definition in the [deepchem library](https://github.com/deepchem/deepchem/blob/master/deepchem/trans/transformers.py#L33)!

If this is correct, then we should be able to remove the `# noqa: F821` and this will be resolved

Reply:
This looks like the right thing to do. I will update this